### PR TITLE
Fix contact page

### PR DESCRIPTION
**Pull request by diamantido**

### CODE REVIEW

File: src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java [hunk 1]
```diff
@@ -57,12 +43,28 @@ protected PasswordAuthentication getPasswordAuthentication() {
         try {
             Message msg = new MimeMessage(session);
             msg.setFrom(new InternetAddress(botEmail, "Harmonizome Bot"));
-            msg.addRecipient(Message.RecipientType.TO, new InternetAddress(avi));
-            msg.addRecipient(Message.RecipientType.CC, new InternetAddress(andrew));
-            msg.addRecipient(Message.RecipientType.CC, new InternetAddress(greg));
+            if(to.isBlank() || to==null) {
```
Comment (u8sand) [inline src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java:46]: You need to test for this to be null before you can use isBlank() otherwise trying to use isBlank() will result in a NullPointerException.

File: src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java [hunk 2]
```diff
@@ -18,34 +15,23 @@ public class Emailer {
...
 	public static void send(String user, String topic, String body) throws UnsupportedEncodingException {
-		final String botEmail = props.getProperty("email.username");
-		final String botPassword = props.getProperty("email.password");
-		String avi = props.getProperty("email.avi");
-		String andrew = props.getProperty("email.andrew");
-		String greg = props.getProperty("email.greg");
+        final String host = System.getenv("SMTP_HOST");
+		final String botEmail = System.getenv("FROM_EMAIL");
+        final String botUser = System.getenv("FROM_USER");
+		final String botPassword = System.getenv("FROM_PASS");
```
Comment (u8sand) [inline src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java:29]: Since these variables do not change they should be `private static final` at the class level.

File: src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java [hunk 3]
```diff
@@ -18,34 +15,23 @@ public class Emailer {
...
+        String to = System.getenv("EMAIL_TO");
+        String cc = System.getenv("EMAIL_CC");
```
Comment (u8sand) [inline src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java:31]: Since these variables do not change they should be `private static final` at the class level.

**Review by u8sand — CHANGES_REQUESTED** [review]
Please see my inline comments

**Review by u8sand — APPROVED** [review]

File: src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java [hunk 4]
```diff
@@ -57,12 +43,28 @@ protected PasswordAuthentication getPasswordAuthentication() {
         try {
             Message msg = new MimeMessage(session);
             msg.setFrom(new InternetAddress(botEmail, "Harmonizome Bot"));
-            msg.addRecipient(Message.RecipientType.TO, new InternetAddress(avi));
-            msg.addRecipient(Message.RecipientType.CC, new InternetAddress(andrew));
-            msg.addRecipient(Message.RecipientType.CC, new InternetAddress(greg));
+            if(to==null || to.isBlank()) {
```
Comment (u8sand) [inline src/main/java/edu/mssm/pharm/maayanlab/Harmonizome/net/Emailer.java:46]: I got the error:
```
09-Feb-2023 18:06:44.997 SEVERE [http-nio-8080-exec-2] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [edu.mssm.pharm.maayanlab.Harmonizome.page.ContactPage] in context with path [/Harmonizome] threw exception [Servlet execution threw an exception] with root cause
        java.lang.NoSuchMethodError: java.lang.String.isBlank()Z
                at edu.mssm.pharm.maayanlab.Harmonizome.net.Emailer.send(Emailer.java:46)
```

in staging..

**Review by u8sand — CHANGES_REQUESTED** [review]
Please see the error I received in staging
